web/themes/custom/imaginelex/templates/paragraphs/paragraph--body-text.html.twig

```diff
@@ -0,0 +1,4 @@
+<h2 class="views--header font-20cen uppercase text-center">
```

Comment:
I think I would rather see the section heading being included here. This is duplicated code that exists in the section-heading atom. It's fine for now but it's the point of why we created the components anyway, right?